templates/openebs-operator-2.6.0.yaml

```diff
@@ -5854,4 +5854,20 @@ spec:
                 - pgrep
                 - "ndo"
             initialDelaySeconds: 30
-            periodSeconds: 60
\ No newline at end of file
+            periodSeconds: 60
+---
+apiVersion: scheduling.k8s.io/v1
+description: Used for system critical pods that must run in the cluster, but can be
+  moved to another node if necessary.
+kind: PriorityClass
+metadata:
+  name: system-cluster-critical
```

Comment:
these priorityclass `system-cluster-critical` and `system-node-critical` are created by the k8s by default and should not be modified , can we remove this and add the custom ones, similar to cstor helm charts